src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -52,6 +56,24 @@
     private FluidRegistry fluidRegistry;
     @In
     private InventoryManager inventoryManager;
+    
+    @In
+    private BlockManager blockManager;
+    private Block air;
+    
+    @In
+    private ExtraBlockDataManager extraDataManager;
+    private int flowIx;
```

Comment:
Oh, it's "index". I called it the same thing in FlowingLiquids's `LiquidFlowSystem` because it's used much more often there. I guess it doesn't really matter here, if you think the longer name would be better.